staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -169,6 +172,10 @@ func (eventBroadcaster *eventBroadcasterImpl) StartRecordingToSink(sink EventSin
 		})
 }
 
+func (eventBroadcaster *eventBroadcasterImpl) Shutdown() {
```

Comment:
this should be `e` instead of `eventBroadcaster `

Reply:
@yastij 
Your comment has been addressed.